src/bindings/samp.h

```diff
@@ -6142,6 +6142,37 @@ static PyObject* pysamp_getvehiclemodelinfo(PyObject *self, PyObject *args)
 	return out;
 }
 
+static PyObject* logprintf_write(PyObject *self, PyObject *args)
+{
+	const char *text;
+
+	if(!PyArg_ParseTuple(args, "es", "cp1252", &text))

```

Comment:
Looks interesting, but I'm not sure about the functionality. Is it possible to set the encoding this way?
This behaviour is not documented in the cpython docs: https://docs.python.org/3/c-api/arg.html#c.PyArg_Parse

Reply:
Do check the "se" specifier (see first response above)